src/Components/Votes/Votes.jsx

```diff
@@ -1,5 +1,31 @@
 import React from "react";
+import { useState } from "react";
+import { updateVotesAdd, updateVotesDec } from "../../utils/API";
+import { Button } from "./Votes.styles";
 
-export default function Votes() {
-  return <div>Votes</div>;
-}
+const Votes = ({ votes, article_id }) => {
+  const [vote, setVote] = useState(0);
+
+  const voteUp = () => {
+    setVote((currentVotes) => currentVotes + 1);
+    updateVotesAdd(article_id).catch();
+  };
+
+  const voteDown = () => {
+    setVote((currentVotes) => currentVotes - 1);
+    updateVotesDec(article_id).catch();
+  };
+
+  console.log(vote, "<<vote", votes, "<<votes");
+  return (
+    <div>
+      <p>
+        Total Votes: {votes + vote}
+        <Button onClick={voteUp}> 🔼 </Button>
```

Comment:
if you are passing an argument to the function on the onClick you will need to wrap it in a function eg 
onClick=(
()=>{functionHere(1)}
)